src/api/routes/accessPass.ts

```diff
@@ -0,0 +1,50 @@
+import { Response } from 'express';
+import { AuthenticatedRequest } from '../AuthenticatedRequest';
+import AsyncRouter from '../AsyncRouter';
+import {
+  createAccessPass,
+  getUser,
+  accessManagerFactory,
+} from '../../application/service';
+import { ApiError } from '../ApiError';
+import { AccessPassFailedError } from '../../application/service/CreateAccessPass';
+
+export default () => {
+  const route = new AsyncRouter();
+
+  route.post(
+    '/users/:id/access-passes',
+    async (req: AuthenticatedRequest, res: Response) => {
+      const { id } = req.params;
+      const { code } = req.body;
+
+      const user = await getUser.byId(id);
```

Comment:
Do we need this part here? Can't we just use the `accessManager.isLoggedInAsUser(new UserId(id))`?

If it returns true then for sure the user exists. if it returns false we anyway would send 404.

Reply:
I think I started this before accessManager existed, but I can switch to it 👍